app/tests.py

```diff
@@ -1,3 +1,11 @@
-from django.test import TestCase
-
+from django.test import TestCase, Client
```

Comment:
nit: you should take care of the order of imports. Check here for PEP recommendation https://www.python.org/dev/peps/pep-0008/#imports